src/draggable.ts

```diff
@@ -108,4 +198,16 @@ export class Draggable extends EventTarget {
 
     this.trigger('move', p);
   }
+
+  resetPosition() {
+    // TODO: Disable pointer events during animation
+    animate((p: number) => {
+      const currentPos = Point.interpolate(this.position, this.startPos, p);
```

Comment:
I'm not sure if this works: `this.position` will change while the animation runs. I think we need to save its initial value in a local variable when `resetPosition` is first called.